spec/order_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'order'
+
+describe Order do
+
+  let(:order) { described_class.new }
+  let(:dish) { "Battered cod" }
+  let(:price) { 3.99 }
```

Comment:
So this is where I think they want us to use doubles, which is what I tried to do. So I THINK if you did let(:dish){double}, it SHOULD give you a double that you can then use in testing. Because then you're keeping the information from the menu class separate from the order class....it's just actually then making it work in testing that gets me!